python/visualization.py

```diff
@@ -615,6 +629,33 @@ def plot_eps(
                 colorbar_parameters=colorbar_parameters,
             )
 
+        if label_geometry:
+            for el in sim.geometry:
+                if sim_size.x == 0:
+                    center_first = el.center.y
+                    center_second = el.center.z
+                    sim_first = sim_center.y
+                    sim_second = sim_center.z
+                elif sim_size.y == 0:
+                    center_first = el.center.y
+                    center_second = el.center.z
+                    sim_first = sim_center.y
+                    sim_second = sim_center.z
+                elif sim_size.z == 0:
+                    center_first = el.center.x
+                    center_second = el.center.y
+                    sim_first = sim_center.x
+                    sim_second = sim_center.y
```

Comment:
Seems like we should have a function for this kind of coordinate projection?

Reply:
I didn't find any helper function to get this done, in `plot_volume` similar structure exist